playbooks/roles/airship-deploy-osh/tasks/main.yml

```diff
@@ -186,7 +186,9 @@
         SHIPYARD_IMAGE: "{{ shipyard_image }}"
         OS_PASSWORD: "{{ lookup('password', secrets_location + '/ucp_shipyard_keystone_password ' + password_opts) }}"
       register: shipyard_desc_action
-      failed_when: "'failed' in shipyard_desc_action.stdout"
+      failed_when: >
+          "'failed' in shipyard_desc_action.stdout" or
+          "'Error' in·shipyard_desc_action.stdout"
```

Comment:
_in·shipyard_desc_action.stdout_ ? What's after _in_ ?

Reply:
oops, that's 'vi' showing blank spaces. Should have used emacs. Fixing.

Reply:
> For that we should modify the until I guess

That until raises my eyebrows, there is no test to find in the stdout that things were fine? Just the absence of "running" or "Processing"? That's weird.

Reply:
> For that we should modify the until I guess

That until raises my eyebrows, there is no test to find in the stdout that things were fine? Just the absence of "running" or "Processing"? That's weird.